app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: ["Distance", int, float]) -> object:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        else:
+            return Distance(km=self.km + other)
+
+    def __iadd__(self, other: ["Distance", int, float]) -> object:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return self
+        else:
+            self.km = self.km + other
+            return self
+
+    def __mul__(self, other: [int, float]) -> object:
+        return Distance(km=self.km * other)
+
+    def __truediv__(self, other: [int, float]) -> object:
+        if other != 0:
+            res_km = round((self.km / other), 2)
+            return Distance(km=res_km)
+
+    def __lt__(self, other: ["Distance", float, int]) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km < other
+        else:
+            return self.km < other.km
+
+    def __gt__(self, other: ["Distance", float, int]) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km > other
+        else:
+            return self.km > other.km
+
+    def __eq__(self, other: ["Distance", int, float]) -> bool:
+        if isinstance(other, (int, float)):
+            return self.km == other
+        else:
+            return self.km == other.km
+
+    def __le__(self, other: ["Distance", int, float]) -> bool:
```

Comment:
You can use previously created magic method here(and in ge as well) to simplify your code. e.g. return not self > other